remme/tp/node_account.py

```diff
@@ -233,6 +235,11 @@ def _transfer_from_frozen_to_unfrozen(self, context, public_key, payload):
         if node_account.node_state != NodeAccount.OPENED:
             raise InvalidTransaction('Masternode is not opened or has been closed.')
 
+        now = datetime.utcnow()
```

Comment:
This will fail transaction validation after a certain amount of time. For example, when a new node is syncing and trying to validate the transaction, this timestamp will be different. Please use the block time from `BlockInfo` instead.

Reply:
But it is wrong, we required to have `last_defrost_timestamp`, which must be now (not block timestamp) and check the current time before we do defrost.